java/libphonenumber/src/com/google/i18n/phonenumbers/metadata/source/MultiFileModeFileNameProvider.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2022 The Libphonenumber Authors
```

Comment:
Can we make class name consistent with the internal one?
SuffixPhoneMetadataFileNameProvider ?

Reply:
Did you check the latest version of the name in internal? It's the same.